src/App.js

```diff
@@ -1,15 +1,38 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList'
+// import Todo from './components/TodoComponents/Todo'
+import TodoForm from './components/TodoComponents/TodoForm'
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      listArray: [],
+    };
+  }
```

Comment:
I'd recommend changing `listArray` into an array of objects (so that you can toggle the completed status of these todos in Day 2, like so:
```
listArray: [
  {
    text: "do something",
    completed: false
  },
  {
    text: "finish something",
    completed: true
  }
]
```

Another recommendation would be to add a prop called `inputText` in your state (so that you can track whatis being typed in the todo form), like so:
```
this.state = {
  inputText:: ""
}
```